### ISSUE TITLE
Not able to disable ember/no-observers in code

### ISSUE THREAD

**Issue by Samsinite** [ember-cli/eslint-plugin-ember#675]
The following code does not seem to disable the `ember/no-observers` rule:

```js
  /* eslint-disable ember/no-observers */
  someObserver: observer('someProperty', function() {
    //...
  }),
  /* eslint-enable ember/no-observers */
```

Since there are specific instances where observers are still useful, seems like a useful rule to be able to disable on a case-by-case basis.

**Comment by Samsinite** [comment]
Single line disabling does not seem to work either on observers. Disabling other rules does work.

**Comment by Samsinite** [comment]
Nevermind, it started flagging the `import { observer } from '@ember/object';` and I didn't read the line number correctly.
